src/Tribe/Ignored_Events.php

```diff
@@ -481,13 +481,101 @@ public function has_ignored_posts( $check_legacy = true ) {
 				return true;
 			}
 
-			$query = new WP_Query( array(
-				'fields' => 'ids',
+			$query = $this->get_ignored_posts_query( array( 'fields' => 'ids' ) );
+
+			return $query->have_posts();
+		}
+
+		/**
+		 * Gets all ids for events that have been ignored
+		 *
+		 * @param null|array $data Array of event IDs/objects/data to check
+		 *
+		 * @return array
+		 */
+		public function get_ignored_posts( $args = array() ) {
+			$query = $this->get_ignored_posts_query( $args );
+
+			return $query->posts;
+		}
+
+		/**
+		 * Gets all ids for events that have been ignored
+		 *
+		 * @param null|array $data Array of event IDs/objects/data to check
+		 *
+		 * @return array
+		 */
+		protected function get_ignored_posts_query( $args = array() ) {
```

Comment:
Lets make this public, no need to call it `get_ignored` because of the class name. We can use `query_posts` no?